python/kserve/pyproject.toml

```diff
@@ -42,7 +42,7 @@ kubernetes = ">=23.3.0"
 python-dateutil = "^2.8.0"
 numpy = "^1.23.5"
 psutil = "^5.9.0"
-ray = { version = "~2.4.0", extras = ["serve"] }
+ray = {version = "~2.4.0", extras = ["serve"]}
```

Comment:
I would leave this line untouched since we are not upgrading ray in this PR

Reply:
I reverted.